server/server.go

```diff
@@ -4,21 +4,23 @@ import (
 	"crypto/rsa"
 	"crypto/tls"
 	"crypto/x509"
-	"github.com/crewjam/saml"
-	"github.com/crewjam/saml/samlsp"
-	"github.com/pkg/errors"
+	"encoding/xml"
 	"io/ioutil"
 	"log"
 	"net/http"
 	"net/url"
+
```

Comment:
Interesting technique. I didn't realize go fmt also honored the imports around blank line separators.